Doc/library/tempfile.rst

```diff
@@ -225,8 +225,10 @@ The module defines the following user-callable items:
    properly implements the :const:`os.O_EXCL` flag for :func:`os.open`.  The
    file is readable and writable only by the creating user ID.  If the
    platform uses permission bits to indicate whether a file is executable,
-   the file is executable by no one.  The file descriptor is not inherited
-   by child processes.
+   the file is executable by no one.
+
+   The file descriptor is not inherited by child processes across
+   :func:`exec <os.execl>` calls.
```

Comment:
I suggested to add a link to https://docs.python.org/dev/library/os.html#fd-inheritance and clarify https://docs.python.org/dev/library/os.html#fd-inheritance about fork vs exec.